docs/install-and-setup.md

```diff
@@ -91,15 +91,13 @@ should never need or want to edit these.
 ## Activate API for the script
 
 Now that the script is saved in your Google Drive folder we need to activate it.
-To do so click the menu `Resources->Advanced Google services`.  
+To do so click the menu `Resources->Advanced Google services`.
 In the popup which will open set "Calendar API" to `enabled` (click the switch
-on its row on the right) and press "Okay".  
-Once you have done this open `Resources->Advanced Google services` again and
-click on the link which says "Google Cloud Platform API Dashboard": you will be
-taken to another page. Click on "Library" in the menu on the left, search for
-"Google Calendar API" in the search box and open it. Now click `Enable` and
-close this page. There is no need to create any credentials as suggested on the
-page.  
+on its row on the right) and press "Okay".
```

Comment:
i can't see what was modified on this line, but shouldn't it be ended with double space? for me there should be a line break at the end